addons/sale/wizard/res_config_settings.py

```diff
@@ -82,11 +75,6 @@ class ResConfigSettings(models.TransientModel):
 
     #=== ONCHANGE METHODS ===#
 
-    @api.onchange('use_quotation_validity_days')
-    def _onchange_use_quotation_validity_days(self):
-        if self.quotation_validity_days <= 0:
-            self.quotation_validity_days = self.env['res.company'].default_get(['quotation_validity_days'])['quotation_validity_days']
-
```

Comment:
L80: `if self.quotation_validity_days < 0:` ?
And adapt the warning message?
Right now, we can't set the default quotation validity to 0 in the setting, which is quite sad